requirements-dev.txt

```diff
@@ -1,7 +1,6 @@
 # This file is auto-generated by scripts/generate_pip_deps_from_conda.py, do not modify.
 # See that file for comments about the need/usage of each dependency.
 
-h5py>=2.7
```

Comment:
why remove this?

Reply:
🤔 the check might need updating here, as h5py is in the other conda environments but not in the windows one

---

or just remove requirements-dev.txt and require pymc devs to use conda for development, that might be simpler 😄

Reply:
I though this was already a dependency in ArviZ, but it looks like that's not the case.
I will undo that change,